app/plugins/Wallet/css/wallet.css

```diff
@@ -218,9 +218,15 @@
 }
 .entry .address {
 	display: inline-block;
-	width: 80%;
+	width: calc(80% - 25px);
```

Comment:
not a good js dev, but is this potentially going to cause problems on smaller or larger screens?